lib/computer.rb

```diff
@@ -0,0 +1,76 @@
+class Computer
```

Comment:
That was fast. You should probably add a module to refactoring list once you know how to do a module. Lots of copying.